.github/workflows/release.yaml

```diff
@@ -174,6 +175,68 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: make -C tools/hashicorp ami
 
+  km-aws-ecs-ami:
+    name: Create AWS ECS AMI
+    runs-on: ubuntu-20.04
+    needs: [km-kkm-smoke-test, km-runenv]
+    if: ${{ github.event_name == 'push' }}
+    steps:
+      - name: Get RELEASE_TAG
+        if: startsWith(github.ref, 'refs/tags/v')
+        run: echo "RELEASE_TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```

Comment:
Not specific to this PR - I wonder if this can be done once for the whole file, not per job

Reply:
we need to checkout first , so maybe we can put it into env and it will be available for all jobs going forward. Need to check on that